build/build.sh

```diff
@@ -90,10 +92,15 @@ if [[ "${BINUTILS_VERSION}" == "trunk" ]]; then
     BINUTILS_NEEDS_GMP=yes
 fi
 
-GCC_REVISION=$(git ls-remote --heads ${URL} refs/heads/${BRANCH} | cut -f 1)
+GCC_REVISION=$(git ls-remote --heads ${URL} "refs/heads/${BRANCH}" | cut -f 1)
 REVISION="gcc-${GCC_REVISION}-binutils-${BINUTILS_REVISION}"
 LAST_REVISION="${3}"
 
+PKGVERSION="Compiler-Explorer-Build"
+if [[ -n "${INCLUDE_SHA1_IN_VERSION_STRING}" ]]; then
+    PKGVERSION+="-${GCC_REVISION}"
```

Comment:
Honestly, there doesn't seem to be a downside to unconditionally enabling this.

Let's just pull the trigger and do it always.

Also, maybe use REVISION instead of GCC_REVISION so we also pick up binutils hashes.

Reply:
ok thanks! Doing it now!